### ISSUE TITLE
[A11y]Screenreader reads as Cancel button expanded which is incorrect as the button is not in expanded state.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8496]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1296231

​## Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Select “API Keys” from the Profile Dropdown and hit enter.
"API Keys" screen gets opened.
Start screenreader and navigate to cancel button and observe screenreader announce state as expanded.
## Actual:​
Screenreader reads as Cancel button expanded which is incorrect as the button is not in expanded state
## Expected:​
Screenreader should not announce as Cancel button expanded as the button is not in expanded state.
​## User Impact:​
Screenreader user will not get proper information regarding the state of the controls.

### Notes
The button functionally behaves as an expander, so I believe the current behaviour (read "expanded") is correct, but we should get more information here.
